draft-ietf-core-coap-tcp-tls.md

```diff
@@ -261,58 +231,40 @@ with a framing mechanism on top of the byte stream provided by TCP/TLS,
 conveying the length information for each message that on datagram transports
 is provided by the UDP/DTLS datagram layer.
 
-TCP ensures reliable message transmission, so the CoAP TCP/TLS messaging
-layer is not required to support acknowledgements or detection of duplicate

```

Comment:
I also struggled with how frequently to reiterate this point, but I agree that we can keep it in this case. I won't take this merge.